providers/base/units/audio/jobs.pxu

```diff
@@ -388,7 +388,7 @@ estimated_duration: 1.0
 imports: from com.canonical.plainbox import manifest
 requires:
   package.name == 'pulseaudio-utils'
-  manifest.has_audio_capture == 'True'
+  audio_card.Capture == 'supported'
```

Comment:
same as above, also, I think that resource/manifest entity names are usually lower case

Reply:
fixed!